src/icon-manifest/manifestMerger.ts

```diff
@@ -87,7 +87,7 @@ export function toggleAngularPreset(
   disable: boolean,
   files: models.IFileCollection): models.IFileCollection {
   const icons = files.supported
-    .filter(x => x.icon.startsWith('ng_') && (x.icon.endsWith('_js') || x.icon.endsWith('_ts')))
+    .filter(x => x.icon.startsWith('ng_') && !x.icon.endsWith('2'))
```

Comment:
This is to allow any extension to be inserted into the angular preset functionality. Before, it was limited to extensions ending in `_js` and `_ts` in order to avoid the activation of `_js2` and `_ts2` icon sets.

As we're willing to allow people to add icons ending in `html, css, less, sass` this restriction has to be more open.